src/main/java/org/humancellatlas/ingest/archiving/submission/ArchiveSubmission.java

```diff
@@ -0,0 +1,34 @@
+package org.humancellatlas.ingest.archiving.submission;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import lombok.Getter;
+import lombok.RequiredArgsConstructor;
+import lombok.Setter;
+import org.springframework.data.annotation.CreatedDate;
+import org.springframework.data.annotation.Id;
+import org.springframework.data.mongodb.core.mapping.Document;
+import org.springframework.hateoas.Identifiable;
+
+import java.time.Instant;
+
+@Getter
+@Document
+@RequiredArgsConstructor
+public class ArchiveSubmission implements Identifiable<String> {
+
+    @Id
+    @JsonIgnore
+    private String id;
+
+    @CreatedDate
+    private Instant created;
+
+    @Setter
+    private String dspUuid;
+
+    @Setter
+    private String dspUrl;
```

Comment:
Could use `URI` here

Reply:
Thanks for the review! yes, i think storing some archive entity information will be useful.
* metadataUuids & data - would be useful to identify which archived entity needs updating
* accession - since we're not patching it back to the json, might be good to store it here * before that
* dsp url & data - could be useful if we need to override the converted data
* dsp uuid - just to easily query it